templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -10,7 +10,7 @@ ENV['EX_DEV_CLIENT_NETWORK_INSPECTOR'] = podfile_properties['EX_DEV_CLIENT_NETWO
 use_autolinking_method_symbol = ('use' + '_native' + '_modules!').to_sym
 origin_autolinking_method = self.method(use_autolinking_method_symbol)
 self.define_singleton_method(use_autolinking_method_symbol) do |*args|
-  if ENV['EXPO_UNSTABLE_CORE_AUTOLINKING'] == '1'
```

Comment:
we can remove `EXPO_UNSTABLE_CORE_AUTOLINKING` and the block entirely. given we don't have rnc-cli installed from 0.76

Reply:
removed